package.json

```diff
@@ -1,18 +1,24 @@
 {
   "name": "nexus-fusion",
   "description": "The interface of Blue Brain Nexus, the open-source knowledge graph for data-driven science.",
-  "version": "1.9.1",
+  "version": "1.10.x",
   "author": "Blue Brain project",
   "license": "Apache-2.0",
   "main": "dist/server.js",
   "repository": "github:bluebrain/nexus-explorer",
   "url": "https://github.com/BlueBrain/nexus/issues/new?labels=fusion",
+  "type": "module",
+  "engine": {
+    "node": ">=18.19.0 <=20.10.0",
+    "npm": ">=9.8.1",
+    "yarn": "1.22.x"
+  },
   "scripts": {
     "clean": "rm -rf ./dist",
     "dev:client": "yarn clean && concurrently 'vite build -c ./vite.config.refresh.js' 'vite'",
-    "dev:server": "NODE_ENV=development ts-node-dev --project tsconfig.node.json -- ./server/index.ts",
+    "dev:server": "NODE_ENV=development tsx --tsconfig ./tsconfig.node.json ./server/index.ts",
     "dev": "concurrently 'yarn dev:client' 'yarn dev:server'",
-    "build:server": "etsc --config=etsc.config.js",
+    "build:server": "etsc --config=etsc.config.cjs",
```

Comment:
Question: Why did you change the config to a CommonJS file? Was there a particular reason for it?